examples/specs/interactive_bar_full_height_hover.vl.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v5.json",
+  "data": {"url": "data/movies.json"},
+  "layer": [
+    {
+      "params": [
+        {
+          "name": "hover",
+          "select": {"type": "point", "on": "mouseover", "clear": "mouseout"}
+        }
+      ],
+      "mark": {"type": "bar", "color": "#eee", "tooltip": true},
+      "encoding": {
+        "x": {"bin": true, "field": "IMDB Rating"},
+        "opacity": {
+          "condition": {"test": {"param": "hover", "empty": false}, "value": 0.5},
+          "value": 0
+        },
+        "detail": [{"aggregate": "count"}]
```

Comment:
This feels odd to me to prevent many bars per x. I wonder whether a distinct transform would be cleaner.

Reply:
I don't understand your point here.  
This is a histogram, so there is only one bar per x and I need to add this line, so the count shows up on the tooltip.

Reply:
Now that I say it, I'll rename the example name to histogram.

Reply:
Oh I see. I didn't realize that was for the tooltip. All good then.